drainer/config.go

```diff
@@ -439,8 +439,11 @@ func (cfg *Config) adjustConfig() error {
 	}
 
 	if cfg.SyncerCfg.DestDBType == "kafka" {
-		maxMsgSize = maxKafkaMsgSize
-
+		if cfg.SyncerCfg.To.KafkaMaxMessageSize > 0 {
+			maxMsgSize = cfg.SyncerCfg.To.KafkaMaxMessageSize
```

Comment:
What if `KafkaMaxMessageSize` > MaxInt32 。 I think we need to do this check here